src/classlibnative/bcltype/system.h

```diff
@@ -38,6 +51,13 @@ class SystemNative
 
 public:
     // Functions on the System.Environment class
+#ifndef FEATURE_PAL
+    static BOOL QCALLTYPE IsLeapSecondsSupportedSystem();
+    static FCDECL1(VOID, GetSystemTimeWithLeapSecondsHandling, FullSystemTime *time);
+    static FCDECL2(FC_BOOL_RET, ValidateSystemTime, FullSystemTime *time, CLR_BOOL localTime);
+    static FCDECL2(FC_BOOL_RET, SystemFileTimeToSystemTime, INT64 fileTime, FullSystemTime *time);
+    static FCDECL2(FC_BOOL_RET, SystemNative::SystemTimeToSystemFileTime, FullSystemTime *time, INT64 *pFileTime);
```

Comment:
` SystemNative::` is not necessary here. (And I believe it will cause warning/error with some compilers.)